code/game/jobs/job/shipside.dm

```diff
@@ -0,0 +1,665 @@
+var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
+
+/datum/job/command
+	department_flag = ROLEGROUP_MARINE_COMMAND
+	selection_color = "#ddddff"
+	supervisors = "the acting commander"
+	idtype = /obj/item/card/id/silver
+	total_positions = 1
+	spawn_positions = 1
+	flags_startup_parameters = ROLE_ADD_TO_DEFAULT
+
+
+//Commander
+/datum/job/command/commander
+	title = "Commander"
+	comm_title = "CO"
+	paygrade = "O4"
+	flag = ROLE_COMMANDING_OFFICER
+	supervisors = "USCM high command"
+	selection_color = "#ccccff"
+	idtype = /obj/item/card/id/gold
+	minimal_player_age = 7
+	flags_startup_parameters = ROLE_ADD_TO_DEFAULT|ROLE_ADD_TO_MODE|ROLE_ADMIN_NOTIFY
+	flags_whitelist = WHITELIST_COMMANDER
+	skills_type = /datum/skills/commander
+	equipment = TRUE
+
+	generate_entry_message()
+		. = {"As the commander of the USS Almayer you are held by higher standard and are expected to act competently.
+While you may support Weyland-Yutani, you report to the USCM High Command, not the corporate office.
+Your primary task is the safety of the ship and her crew, and ensuring the survival and success of the marines.
+Your first order of business should be briefing the marines on the mission they are about to undertake.
+If you require any help, use adminhelp to ask mentors about what you're supposed to do.
+Godspeed, commander! And remember, you are not above the law."}
+
+	announce_entry_message(mob/living/carbon/human/H)
```

Comment:
`/datum/job/command/announce_entry_message(mob/living/carbon/human/H)`
And then fix indentation.